### PR TITLE
Feature/related fields

### PR DESCRIPTION
**Pull request by flibbertigibbet**

### CODE REVIEW

**Comment by ddohler** [comment]
I'm not able to test at the moment because I'm building HunchLab AMIs, but once that finishes I'll take this for a spin. Code looks good though.

File: app/src/main/java/org/worldbank/transport/driver/activities/RecordFormActivity.java [hunk 1]
```diff
@@ -292,42 +287,25 @@ private FormSectionController addSectionModel() {
...
 
-                        Log.d(LOG_LABEL, "enumLabels: " + enumLabels.toString());
-                        Log.d(LOG_LABEL, "enumValues: " + enumValueObjectList.toString());
-
-                        // TODO: no matter what gets passed for the prompt argument, it seems to always display "Select"
-                        control = new SelectionController(this, fieldName, fieldLabel, isRequired, "Select", enumLabels, enumValueObjectList);
+                        // TODO: fix or remove prompt arg in form builder library
+                        // no matter what gets passed for the prompt argument, it seems to always display "Select"
+                        control = new SelectionController(this, fieldName, fieldLabel, isRequired, "Select",
+                                enumListInfo.labels, enumListInfo.items);
```
Comment (ddohler) [inline app/src/main/java/org/worldbank/transport/driver/activities/RecordFormActivity.java:295]: The indentation here looks a bit off.

**Comment by ddohler** [comment]
+1 I tested this locally, looks good!

File: app/src/main/java/org/worldbank/transport/driver/activities/RecordFormActivity.java (hunk 1 above)
Comment (flibbertigibbet) [inline app/src/main/java/org/worldbank/transport/driver/activities/RecordFormActivity.java:295]: That's what IntelliJ does with it. Could leave it on one giant line instead.

Comment (ddohler) [inline app/src/main/java/org/worldbank/transport/driver/activities/RecordFormActivity.java:295]: If this is the IntelliJ default then probably best to leave it be, at least it'll be consistent.

**Pull request merged**